### PR TITLE
Release notes 114

### PR DESCRIPTION
**Pull request by meagancojocar**

### CODE REVIEW

File: content/blog/pulumi-release-notes-114/index.md [hunk 1]
```diff
@@ -0,0 +1,117 @@
...
+
+## Pulumi Environments, Secrets and Configuration (Pulumi ESC)
+
+### General Availability & Key Enhancements
+
+- **[ESC General Availability](/blog/pulumi-esc-ga/)**  
+  GA of Pulumi ESC -redefining secrets management with features like dynamic credentials, hierarchical environments, a secrets broker with external stores like AWS Secrets Manager, and Kubernetes operators for runtime integration.
+- **[ESC Projects & Environment Tags](/blog/esc-projects-environment-tags-launch/)**  
+  Organize secrets and configurations at scale with new grouping capabilities.
+- **[ESC Webhooks](/blog/esc-webhooks-launch/)**
```
Comment (cleverguy25) [inline content/blog/pulumi-release-notes-114/index.md:80]: Can we add Parameter store and Imported By blogs

File: content/blog/pulumi-release-notes-114/index.md [hunk 2]
```diff
@@ -0,0 +1,117 @@
...
+  Manage explicit dependencies between provider functions for more accurate execution in complex deployments.
+
+---
+
+## Pulumi Environments, Secrets and Configuration (Pulumi ESC)
+
+### General Availability & Key Enhancements
+
+- **[ESC General Availability](/blog/pulumi-esc-ga/)**  
+  GA of Pulumi ESC -redefining secrets management with features like dynamic credentials, hierarchical environments, a secrets broker with external stores like AWS Secrets Manager, and Kubernetes operators for runtime integration.
```
Comment (thtmnisamnstr) [inline content/blog/pulumi-release-notes-114/index.md:77]:
```suggestion
  GA of Pulumi ESC - redefining secrets management with features like dynamic credentials, hierarchical environments, a secrets broker with external stores like AWS Secrets Manager, and Kubernetes operators for runtime integration.
```

File: content/blog/pulumi-release-notes-114/index.md [hunk 3]
```diff
@@ -0,0 +1,117 @@
...
+## Pulumi AI and Insights
+
+### Pulumi Insights 2.0
+
+- **[Resource Explorer](/blog/insights-resources-v2/)**  
+  Enhanced visual interface for exploring cloud resources with new features like customizable columns, nested grouping capabilities, advanced filtering, and shareable saved views across your organization.
+- **[Account Discovery Preview](/blog/insights-cloud-account-discovery/)**  
+  New capability to discover and manage all cloud resources regardless of how they were created, with AI-powered search and comprehensive resource visibility across your infrastructure.
+
+### Pulumi Pulumi Copilot Enhancements
```
Comment (thtmnisamnstr) [inline content/blog/pulumi-release-notes-114/index.md:105]:
```suggestion
### Pulumi Copilot Enhancements
```

**Review by thtmnisamnstr — APPROVED** [review]

File: content/blog/pulumi-release-notes-114/index.md (hunk 1 above)
Comment (meagancojocar) [reply content/blog/pulumi-release-notes-114/index.md:80]: done!

**Pull request merged**